pydatastructs/graphs/algorithms.py

```diff
@@ -11,6 +11,7 @@
 from pydatastructs.graphs.graph import Graph
 from pydatastructs.linear_data_structures.algorithms import merge_sort_parallel
 from pydatastructs import PriorityQueue
+import math
```

Comment:
Is importing math necessary for binary lifting?

Reply:
Yes I suppose. I have used the log2 function from the math library in my code.